src/Rules/Resources/HowToFix.Designer.cs

```diff
@@ -180,7 +180,7 @@ internal class HowToFix {
         }
         
         /// <summary>
-        ///   Looks up a localized string similar to A button element should only support the one of Invoke, Toggle, or ExpandCollapse patterns when a splitbutton is the parent. .
+        ///   Looks up a localized string similar to A button element should only support one of Invoke, Toggle, or ExpandCollapse patterns when a splitbutton is the parent. .
```

Comment:
Perhaps a bit pedantic, but I'd include a "the":  ... should only one of **the** Invoke, Toggle, or ExpandCollapse patterns ...